Ruby/book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author
+
+  @rental = Rental
+  def intialize(title, author)
+    @title = title
+    @author = author
+  end
```

Comment:
- [ ] It would be better if we have a method for adding rentals. Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) that can help us